pkg/lib/util/util.go

```diff
@@ -88,9 +91,70 @@ func Xhtml2Json(rawXhtml []byte) ([]byte, error) {
 		return []byte{}, errors.Wrap(err, "Could not parse xhtml")
 	}
 
-	json, err := mv.JsonIndent("", " ")
+	jsonStr, err := mv.JsonIndent("", " ")
 	if err != nil {
 		return []byte{}, errors.Wrap(err, "Could not convert to json")
 	}
-	return json, nil
+	return jsonStr, nil
+}
+
+// Html2Json parses the raw html data and converts it into a json string
+func Html2Json(rawHtml []byte) ([]byte, error) {
+	var (
+		htmlDoc *goquery.Document
+		err     error
+	)
+
+	htmlDoc, err = goquery.NewDocumentFromReader(bytes.NewReader(rawHtml))
+	if err != nil {
+		return []byte{}, errors.Wrap(err, "Could not parse html")
+	}
+
+	htmlData := parseHtmlNode(htmlDoc.Selection)
+	jsonStr, err := json.MarshalIndent(htmlData, "", " ")
```

Comment:
Use "golib.JsonIndent" here. It does a better job as it is not encoding HTML special chars. That's a shitty default in Go.

Reply:
`golib.JsonIndent` was ist das für ein package / für eine Funktion? Mein go compiler kennt das nicht und google findet dazu nichts

Reply:
sorry das heisst `golib.JsonBytesIndent`. such mal im code. package "github.com/programmfabrik/golib"

Reply:
es gibt auch noch JsonStringIndent...

Reply:
Das apitesttool hatte noch veraltete dependencies, hab die `go.mod` auch auf `go 1.20` aktualisiert, jetzt ist das die selbe version wie fylr.